src/allmydata/scripts/cli.py

```diff
@@ -90,6 +93,7 @@ class ListOptions(FilesystemOptions):
         ("readonly-uri", None, "Show read-only file/directory URIs."),
         ("classify", "F", "Append '/' to directory names, and '*' to mutable."),
         ("json", None, "Show the raw JSON output."),
+        ("directory", "d", "Show the directory itself, not content."),
```

Comment:
i believe this conflicts with the tahoe global option::
  -d, --node-directory=